.buildkite/pipeline.yml

```diff
@@ -41,6 +41,16 @@ steps:
       provider: "gcp"
       machineType: "n1-standard-8"
 
+  - label: "Serverless integration test"
+    key: "serverless-integration-tests"
+    command: ".buildkite/scripts/steps/integration_tests.sh serverless integration:single TestMonitoringLogsShipped"
```

Comment:
Are we hardcoding a single test here?

Reply:
Yah. @cmacknz 's idea was to just run a single high-level test for serverless, so we can have some coverage without running the entire test suite twice.

Reply:
Correct, this is intended to be a basic sanity check since nothing in agent itself has to change to support serverless (besides a single additional HTTP header).

We don't need to run the entire 1+ hour test suite a second time for this.

I think a clarifying comment in this file explaining this should be added.

Reply:
Thanks for the clarification folks, it totally makes sense. A comment clarifying it would be great.